tests/unit/test_ingest_stripe.py

```diff
@@ -653,9 +650,10 @@ def test_get_email_id_subscription(
 
 
 def test_get_email_id_invoice(
-    dbsession, contact_with_stripe_customer, raw_stripe_invoice_data
+    dbsession, stripe_customer_factory, raw_stripe_invoice_data
 ):
     """A Stripe Invoice and related objects can return the related email_id."""
+    customer = stripe_customer_factory(stripe_id=FAKE_STRIPE_CUSTOMER_ID)
```

Comment:
Why do we set the id here?

Reply:
While we're in the middle of conversion, there are still a few places where we have "magic" object IDs. In this case, the way we use `raw_stripe_invoice_data` assumes that there's a a customer whose ID is `FAKE_STRIPE_CUSTOMER_ID`. So for now, that's what we specify when we generate the Stripe customer, but in a follow-up PR, we'll hopefully be able to remove this.